ksql-engine/src/test/java/io/confluent/ksql/integration/WindowingIntTest.java

```diff
@@ -262,7 +262,7 @@ private void assertTableCanBeUsedAsSource(
   private void assertTopicsCleanedUp(final TopicCleanupPolicy topicCleanupPolicy) {
     assertThat("Initial topics", getTopicNames(), hasSize(5));
 
-    ksqlContext.getRunningQueries().forEach(QueryMetadata::close);
+    ksqlContext.getPersistentQueries().forEach(QueryMetadata::close);
```

Comment:
maybe add a method to `KsqlContext` that closes the persistent queries?

Reply:
Don't think its worth it just for a test.